test/services/TaxEnrolmentsServiceSpec.scala

```diff
@@ -38,40 +38,43 @@ class TaxEnrolmentsServiceSpec extends BaseSpec {
     reset(mockConnector)
   }
 
-  ".setSubscriptionId" should {
+  ".setSubscriptionId 4MLD" should {
+
+    val taxable: Boolean = false
+    val trn = "XTRN1234567"
 
     "return TaxEnrolmentSuccess  " when {
       "connector returns success taxEnrolmentSuscriberResponse." in {
-        when(mockConnector.enrolSubscriber("123456789")).
+        when(mockConnector.enrolSubscriber("123456789", taxable, trn)).
```

Comment:
Are we missing specs for 5MLD?

Reply:
Updated but getting failed tests when mocking config returns

Reply:
tests updated to pass taxable flag and trn